### DISCUSSION TITLE
Next release planned?

### DISCUSSION THREAD

**Discussion by noomio** [olikraus/u8g2#2019] [General]
Hi,

When are you planning the next release?

Is it OK to use master as stable?

Thanks

**Comment by olikraus** [comment]
Is there anything special you need in the next release?  https://github.com/olikraus/u8g2/milestone/40
And yes, using latest version in https://github.com/olikraus/U8g2_Arduino is always ok.

I think you can do this with PlatformIO:

`platformio.ini` (https://docs.platformio.org/en/latest/projectconf/section_env_library.html#lib-deps) should include 
```
lib_deps =
  u8g2=https://github.com/olikraus/U8g2_Arduino/archive/master.zip
```

**Comment by noomio** [comment]
Ok thanks. I'm using [2.29.11](https://github.com/olikraus/u8g2/releases/tag/2.29.11)

Mostly interested if there are any fixes to bugs reported and always try and keep up to date with the latest release.

I'm not using the the Arduino one.

**Comment by olikraus** [comment]
Ah, ok.
At the moment there is no specific plan for another official release.
